<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>wdgStrategyResults</class>
 <widget class="QWidget" name="wdgStrategyResults">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>656</width>
    <height>497</height>
   </rect>
  </property>
  <property name="windowIcon">
   <iconset resource="../images/xulpymoney.qrc">
    <normaloff>:/xulpymoney/bank.png</normaloff>:/xulpymoney/bank.png</iconset>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="lbl">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Strategies results</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="chkFinished">
       <property name="text">
        <string>Show finished strategies</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="mqtwObjects" name="mqtwStrategies" native="true"/>
     </item>
    </layout>
   </item>
  </layout>
  <action name="actionStrategyNew">
   <property name="icon">
    <iconset resource="../images/xulpymoney.qrc">
     <normaloff>:/xulpymoney/add.png</normaloff>:/xulpymoney/add.png</iconset>
   </property>
   <property name="text">
    <string>New strategy</string>
   </property>
   <property name="toolTip">
    <string>New strategy</string>
   </property>
  </action>
  <action name="actionStrategyDelete">
   <property name="icon">
    <iconset resource="../images/xulpymoney.qrc">
     <normaloff>:/xulpymoney/button_cancel.png</normaloff>:/xulpymoney/button_cancel.png</iconset>
   </property>
   <property name="text">
    <string>Delete strategy</string>
   </property>
   <property name="toolTip">
    <string>Delete strategy</string>
   </property>
  </action>
  <action name="actionStrategyEdit">
   <property name="icon">
    <iconset resource="../images/xulpymoney.qrc">
     <normaloff>:/xulpymoney/editar.png</normaloff>:/xulpymoney/editar.png</iconset>
   </property>
   <property name="text">
    <string>Edit strategy</string>
   </property>
   <property name="toolTip">
    <string>Edit strategy</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>mqtwObjects</class>
   <extends>QWidget</extends>
   <header location="global">myqtablewidget</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../images/xulpymoney.qrc"/>
 </resources>
 <connections/>
</ui>
